lib/ansible/cli/galaxy.py

```diff
@@ -498,6 +498,8 @@ def add_install_options(self, parser, parents=None):
             install_parser.add_argument('--ignore-signature-status-code', dest='ignore_gpg_errors', type=str, action='append',
                                         help=ignore_gpg_status_help, default=C.GALAXY_IGNORE_INVALID_SIGNATURE_STATUS_CODES,
                                         choices=list(GPG_ERROR_MAP.keys()))
+            install_parser.add_argument('--git-init-submodules', dest='git_init_submodules', action='store_true',
```

Comment:
This is only added to the `install` subcomand meaning that in other cases there would be access of a non-existing undefined CLI option which is visible in the CI failures.

Reply:
It can even cause a KeyError when when installing collections, because `ansible-galaxy install` will install roles and collections at the same time but doesn't allow collection-specific cli options. We use `context.CLIARGS.get()` for that issue in a few places. Here's an example: https://github.com/s-hertel/ansible/commit/0bfec37608e0bf9af2911f25e426b9d9175a051b.